src/formio.component.ts

```diff
@@ -52,7 +52,10 @@ export class FormioComponent implements OnInit {
         if (this.form) {
             this.ready.next(true);
         }
-        else if (this.src && !this.service) {
+        else if (this.src) {
+          if (!this.service){
+            this.service = new FormioService(this.src);
+            }
             this.service = new FormioService(this.src);
```

Comment:
I have raised pullrequest [BugFix/ServiceObject #60](https://github.com/formio/ng2-formio/pull/60)Kindly verify code ,consider merging the pull request with your formio master. Please let me know your comments.